lte/gateway/python/magma/subscriberdb/client.py

```diff
@@ -253,49 +253,46 @@ async def _get_all_subscribers(self) -> Optional[CloudSubscribersInfo]:
         return subscribers_info
 
     async def _list_suciprofiles(self) -> Optional[CloudSuciProfilesInfo]:
+        req_page_token = ""
         subscriberdb_cloud_client = self._grpc_client_manager.get_client()
-        suci_profiles = SuciProfile()
-        while not suci_profiles:
-            try:
-                res = await grpc_async_wrapper(
-                    subscriberdb_cloud_client.ListSuciProfiles.future(
-                        self.SUBSCRIBERDB_REQUEST_TIMEOUT,
+        suciprofiles = []
+        try:
+            req = ListSubscribersRequest(
+                page_size=self._subscriber_page_size,
+                page_token=req_page_token,
+            )
+            res = await grpc_async_wrapper(
```

Comment:
Seems ListSuciProfiles rpc does not takes any argument in. Is it required to pass the req parameter.

Reply:
![MicrosoftTeams-image (14)](https://user-images.githubusercontent.com/93516965/171639580-d7a0a2b2-1802-4307-8296-5ae4365f9409.png)
It is required to pass the req parameter without which Traceback is generated.